solution/backend/regulations/views/reader.py

```diff
@@ -53,8 +52,7 @@ def get_context_data(self, **kwargs):
         part_label = toc['label_description']
         tree = self.get_content(context, document, toc)
         node_list = self.get_supp_content_params(context, [tree])
-        categories = list(Category.objects.filter(show_if_empty=True).contains_fr_docs().order_by('order').values())
-        sub_categories = list(SubCategory.objects.filter(show_if_empty=True).contains_fr_docs().order_by('order').values())
+        categories = list(Category.objects.contains_fr_docs().order_by('order').values())
```

Comment:
we're just sending all categories into the template for use on the front end.  That's now where we will decide if empty categories will be shown or hidden